packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -278,6 +279,8 @@ const SceneView = ({
       onNativeDismissCancelled={onNativeDismissCancelled}
       // this prop is available since rn-screens 3.16
       freezeOnBlur={freezeOnBlur}
+      accessibilityElementsHidden={!isFocused}
+      importantForAccessibility={isFocused ? 'auto' : 'no-hide-descendants'}
```

Comment:
Does React Native Screens support these props? I don't see it in their typedefs. I think proper place for these would be the `<View style={styles.scene}>`